cmd/share-secret/storedata.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"time"
+
+	"github.com/google/uuid"
+)
+
+type StoreData struct {
+	Secret     string
+	Unit       string
+	Time       int64
+	Registered *time.Time
+	Expire     *time.Time
+}
+
+func (s *StoreData) GetExpirationDate() *time.Time {
+	return nil
+}
+
+func (s *StoreData) GetShareData() string {
+
+	return ""
+}
+func (s *StoreData) GetKey() string {
```

Comment:
Useless, the `key` should be generated the moment it is stored in the map (or later in a database), and returned to the caller. Don't need this function attached to `StoreData`. It can be an isolated function instead